examples/3d/wireframe.rs

```diff
@@ -56,3 +74,22 @@ fn setup(
         ..default()
     });
 }
+
+/// This timer is used to periodically toggle the wireframe rendering.
+#[derive(Resource)]
+struct WireFrameToggleTimer(Timer);
```

Comment:
Probably should be named like this.
```suggestion
struct WirerameToggleTimer(Timer);
```

Reply:
Do you mean `Wireframe`? Without the capital "f"?

Reply:
Yes, updated :smile: